main-site/components/adventure-ahead-section/AdventureAheadSection.styles.ts

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+import { P } from '../../../shared-ui/style/typography';
+import { min, max } from '../../../shared-ui/lib/responsive';
+import { colors } from '../../../shared-ui/style/colors';
+
+const StyledAdventureAheadSectionContainer = styled.div`
+  padding-top: 16em;
+
+  @media ${min.mobile} {
+    margin-top: 14em;
+  }
+  @media ${min.tabletSm} {
+    margin-top: 24em;
+  }
+  @media ${min.tablet} {
+    margin-top: 4em;
+  }
+  @media ${min.tabletLg} {
+    margin-top: 10em;
+  }
+  @media ${min.desktop} {
+    margin-top: 23em;
+  }
+`;
+
+const StyledParagraph = styled(P)`
+  padding-top: 1em;
+  padding-bottom: 1em;
+  color: ${colors.TEXT_BROWN};
+`;
+const StyledSignpost = styled.img`
+  padding-left: 8em;
+  display: inline-block;
+  vertical-align: top;
+`;
+
+const StyledTextContainer = styled.div`
+    display: inline-block;
+    justify-content: center;
+    color: ${colors.TEXT_BROWN}
+    display: inline-block;
```

Comment:
instead of adding all this inline block stuff, use display: flex in your StyledAdventureAheadSectionContainer instead on desktop.